sdk/nodejs/.eslintrc.js

```diff
@@ -1,84 +1,68 @@
 module.exports = {
-    "env": {
-        "es6": true,
-        "node": true
+    env: {
+        es6: true,
+        node: true,
     },
-    "parser": "@typescript-eslint/parser",
-    "parserOptions": {
-        "project": "tsconfig.json",
-        "tsconfigRootDir": __dirname,
-        "sourceType": "module"
+    parser: "@typescript-eslint/parser",
+    parserOptions: {
+        project: "tsconfig.json",
+        tsconfigRootDir: __dirname,
+        sourceType: "module",
     },
-    "plugins": [
-        "eslint-plugin-import",
-        "@typescript-eslint",
-        "header"
-    ],
-    "ignorePatterns": [
+    plugins: ["eslint-plugin-import", "@typescript-eslint", "header"],
+    ignorePatterns: [
         "**/bin/**/*.ts",
         "tests/automation/data/tcfg/*.ts",
         "tests/sxs_ts_test/*.ts",
         "tests/mockpackage/**/*.ts",
     ],
-    "rules": {
+    rules: {
         "header/header": [
             2,
             "line",
             [
                 {
-                    "pattern": "Copyright \\d{4}-\\d{4}, Pulumi Corporation."
-                }
-            ]
+                    pattern: "Copyright \\d{4}-\\d{4}, Pulumi Corporation.",
+                },
+            ],
         ],
         "@typescript-eslint/dot-notation": "off",
         "@typescript-eslint/explicit-member-accessibility": [
             "off",
             {
-                "accessibility": "explicit"
-            }
-        ],
-        "@typescript-eslint/indent": [
-            "error",
-            4,
-            {
-                "FunctionDeclaration": {
-                    "parameters": "first"
-                },
-                "FunctionExpression": {
-                    "parameters": "first"
-                }
-            }
```

Comment:
This is probably the most "controversial" change. ESLint is doing double duty for us, policing both linting _and_ formatting. This change removes ESLint's "indentation" rule because it's incompatible with Rome's rules on some minor edge cases: switch statement cases indentation, nested ternary operators, etc. Otherwise, the two tools are compatible.